src/include/esp-idf/bindings.h

```diff
@@ -305,6 +305,10 @@
 #include "mqtt_client.h"
 #endif
 
+#ifdef CONFIG_MQTT_PROTOCOL_5
```

Comment:
Move this block _inside_ the above block protected with `ESP_IDF_COMP_MQTT_EMABLED`

Reply:
Done!